src/java/org/apache/cassandra/index/sai/utils/RangeIntersectionIterator.java

```diff
@@ -55,11 +57,14 @@ public static boolean shouldDefer(int numberOfExpressions)
     }
 
     private final List<RangeIterator> ranges;
+    private final int[] rangeStats;
 
     private RangeIntersectionIterator(Builder.Statistics statistics, List<RangeIterator> ranges)
     {
         super(statistics);
         this.ranges = ranges;
+        this.rangeStats = new int[ranges.size()];
+        Arrays.fill(rangeStats, 0);
```

Comment:
This is a no-op since Java arrays are guaranteed to initialize to zero (or null)